RubeesEat.IntegrationTests/TestBases/Selenium/WithTestBills.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Immutable;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using RubeesEat.Model;
+
+namespace RubeesEat.IntegrationTests;
+
+public class WithTestBills : WithTestPersons
```

Comment:
Interesting test base name. I recommend making them abstract. As to naming, I can see it. I hope @a-ctor likes it, too.